src/components/pages/AdvancedApiFilter/AsylumOfficeSelect.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { setAsylumOfficeFilter } from '../../../state/actionCreators';
+import 'antd/dist/antd.css';
+import { Select, Popover } from 'antd';
+import { officeRegions } from '../../../data/filterConstants';
+
+const { Option } = Select;
+
+const mapStateToProps = state => ({
+  asylumOffice: state.fetchFilterReducer.asylumOffice,
+});
+
+const children = officeRegions.map(office => (
+  <Option key={office.code}>
+    <Popover
+      placement="right"
+      title={`${office.location.city}, ${office.location.state}`}
+      content={office.regions.map(region => (
+        <p key={region}>{region}</p>
+      ))}
+      trigger="hover"
+    >
+      <p style={{ textAlign: 'center' }}>
+        {/* {`${office.location.city}, ${office.location.state}`} */}
+        {/*To enable selection of multiple regions, replace the preceding line with the following:*/}
+        {office.location.state}
+      </p>
+    </Popover>
+  </Option>
+));
+
+const AsylumOfficeSelect = ({ asylumOffice, setAsylumOfficeFilter }) => {
+  const dispatchChange = officeArray => {
+    setAsylumOfficeFilter(
+      typeof officeArray === 'string' ? [officeArray] : officeArray
+    );
+  };
+  return (
+    <Select
+      // Uncomment the following line to allow for selection of multiple regions
```

Comment:
Not sure if there was a commented out line that was missing or line 41 was left by accident.

Reply:
That was from when I initially created the component and multi-selection query strings weren't part of the API specs. It was intended to allow our next Bloomtechies to know that they can switch between single and multiple selection modes by swapping out these lines. As multiple selection has become the decided version, this comment can be removed whenever somebody puts in a pull request in the future, if they remember.